### PR TITLE
CNV-3881 New Feature Release Note for SVVP Certification

### PR DESCRIPTION
**Pull request by bgaydosrh**
Label for enterprise-4.4.

Test Build: http://file.bos.redhat.com/bgaydos/042220/cnv/cnv_release_notes/cnv-release-notes.html

Code and QE Review are now complete.
Product Management review and Peer Review from @aburdenthehand are all that are needed.

### CODE REVIEW

File: cnv/cnv_release_notes/cnv-release-notes.adoc [hunk 1]
```diff
@@ -20,6 +20,13 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
 * You can now use {CNVProductName} with either the xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#learn-about-ovn-kubernetes[OVN-Kubernetes] or the xref:../../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[OpenShiftSDN] network provider.
 
+// CNV-3881 Release note to announce SVVP Certification
```
Comment (aburdenthehand) [inline cnv/cnv_release_notes/cnv-release-notes.adoc:23]: @bgaydosrh Can you please move this to replace the placeholder(`// CNV-4310/CNV-3881 SVVP`)  at L48

Comment (bgaydosrh) [reply cnv/cnv_release_notes/cnv-release-notes.adoc:23]: Will do today, @aburdenthehand. Thanks. Bob

Comment (bgaydosrh) [reply cnv/cnv_release_notes/cnv-release-notes.adoc]: Done.

File: cnv/cnv_release_notes/cnv-release-notes.adoc [hunk 2]
```diff
@@ -41,8 +41,12 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
 // CNV-4078 UI pause/unpause
 
-// CNV-4310/CNV-3881 SVVP
-
+// CNV-3881 Release note to announce SVVP Certification
+* {CNVProductNameStart} is certified in Microsoft's Windows Server
+Virtualization Validation Program (SVVP) to run Windows Server workloads.
+** SVVP Certification applies to Intel and AMD CPUs.
+** SVVP Certificate RHCOS Core OS8 workers are named _Red Hat OpenShift
```
Comment (aburdenthehand) [inline cnv/cnv_release_notes/cnv-release-notes.adoc:48]: According to the master product sheet:
"RHCOS Core OS8" should be "Red Hat Enterprise Linux CoreOS 8" at first usage

Comment (bgaydosrh) [reply cnv/cnv_release_notes/cnv-release-notes.adoc:48]: Cool, cool. Nice catch!

**Pull request merged**